### ISSUE TITLE
 local.ERROR: ErrorException: Undefined variable: data

### ISSUE THREAD

**Issue by cielu** [swooletw/laravel-swoole#232]
Make sure you read **Issues Guideline** and answer these questions before submitting your issue. Thanks!
(Any **non-English** issues will be closed immediately.)

1. Please provide your PHP and Swoole version. (`php -v` and `php --ri swoole`)

7.2.3
4.1

2. Please provide your Laravel/Lumen version.

5.7.*

3. Which release version of this package are you using?

2.6.5

4. What did you do? If possible, provide a recipe for reproducing the error.

 local.ERROR: ErrorException: Undefined variable: data in laravel-swoole/src/Server/Manager.php:269
```
    /**
     * Set onTask listener.
     *
     * @param mixed $server
     * @param string|\Swoole\Server\Task $taskIdOrTask
     * @param string $srcWorkerId                       Optional
     * @param mixed $data                               Optional
     */
    public function onTask($server, ...$args)
    {
        $this->container->make('events')->dispatch('swoole.task', [$server, $args]);

        try {
            // push websocket message
            if ($this->isWebsocketPushPayload($data)) {
                $this->pushMessage($server, $data['data']);
            // push async task to queue
            } elseif ($this->isAsyncTaskPayload($data)) {
                (new SwooleTaskJob($this->container, $server, $data, $taskId, $srcWorkerId))->dispatch();
            }
        } catch (Throwable $e) {
            $this->logServerError($e);
        }
    }
```

5. What did you expect to see?

6. What did you see instead?

**Comment by albertcht** [comment]
Hi @cielu ,

Thanks for your report, I will fix this soon.

**Comment by cielu** [comment]
兄弟，不止这一个问题啊

when I connected by socket.io
```
    socket.on('connect', () => {
        console.log('connect the socket')
        socket.emit('example', 'hello example')
    })
```
server code => routes/websocket.php 
```
Websocket::on('example', function ($websocket, $data) {
    Log::debug($data);
    $websocket->emit('other', $data);
});
```
could receive my emit , but I can't emit message back to client

**Comment by albertcht** [comment]
The websocket feature has been fixed in the newest hotfix release `v2.6.5.1`.

**Comment by cielu** [comment]
> The websocket feature has been fixed in the newest hotfix release `v2.6.5.1`.

OK，thanks
